server/handlers.go

```diff
@@ -0,0 +1,113 @@
+package server
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sync"
+
+	"github.com/ASeegull/SmartFridge/server/database"
+	"github.com/gorilla/websocket"
+	log "github.com/sirupsen/logrus"
+)
+
+var wg sync.WaitGroup
+
+func getHash(stringToHash string) string {
+	hash := md5.Sum([]byte(stringToHash))
+	return hex.EncodeToString(hash[:])
+}
+
+func agentAuthentication(w http.ResponseWriter, r *http.Request) {
+	tokenExample := getHash("userID+agentID+productName")
+	json.NewEncoder(w).Encode(tokenExample)
+}
+
+func createWS(w http.ResponseWriter, r *http.Request) {
+	conn, err := upgrader.Upgrade(w, r, nil)
+
+	if err != nil {
+		log.WithField("error: ", err).Info("Problem with creation websocket")
+		return
+	}
+
+	fmt.Printf("New websocket connect with %s\n", r.Host)
+	wg.Add(1)
+	go wsListener(conn)
+}
+
+func wsListener(conn *websocket.Conn) {
+	defer wg.Done()
+	defer conn.Close()
+
+	for {
```

Comment:
Do not use not controlled infinitive loops! You would burn out your CPU.
Add some blocking operation not to fun `for` all the time.